packages/mobile/src/screens/settings-screen/CastSettingsRow.tsx

```diff
@@ -29,7 +29,7 @@ export const CastSettingsRow = () => {
 
   const setCastMethod = useCallback(
     (method: CastMethod) => {
-      dispatchWeb(updateMethod({ method: method }))
+      dispatchWeb(updateMethod({ method: method, persist: true }))
```

Comment:
just curious what the persist is?!

Reply:
will add comment! this is so that it is written to async storage so your preference is "persisted" across sessions